src/main/webapp/js/adminAccountManagement.es6

```diff
@@ -109,24 +108,23 @@ function reLabelOrderedAccountEntries() {
 }
 
 function showEntriesForSelectedPage() {
-    begin = (currentPage - 1) * entryPerPage + 1;
-    end = begin + entryPerPage - 1;
+    begin = (currentPage - 1) * (entryPerPage + 1);
```

Comment:
Doesn't this change the logic?

(although admittedly I don't know where this is used)